tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1122,6 +1123,65 @@ def test_go_must_fail_with_unknown_architecture(self, runtime, code_uri, archite
         self.assertEqual(process_execute.process.returncode, 1)
 
 
+@skipIf(
+    ((IS_WINDOWS and RUNNING_ON_CI) and not CI_OVERRIDE),
+    "Skip build tests on windows when running in CI unless overridden",
+)
+@parameterized_class(
+    ("template", "code_uri", "binary", "expected_invoke_result"),
+    [
+        (
+            "template_build_method_rust_single_function.yaml",
+            "Rust/single-function",
+            None,
+            {"req_id": "34", "msg": "Hello World"},
+        ),
+        (
+            "template_build_method_rust_binary.yaml",
+            "Rust/multi-binaries",
+            "function_a",
+            {"req_id": "63", "msg": "Hello FunctionA"},
+        ),
+        (
+            "template_build_method_rust_binary.yaml",
+            "Rust/multi-binaries",
+            "function_b",
+            {"req_id": "99", "msg": "Hello FunctionB"},
+        ),
+    ],
+)
+class TestBuildCommand_Rust(BuildIntegRustBase):
```

Comment:
Is Rust buildable in a container? If so, should we have a test for that too?

Reply:
It's buildable in a container but customers will need to provide their own build image.
For now, our `provided.al2` build image doesn't have Rust toolchain installed. We have a plan to provide individual `provided.al2` build images with specific toolchain for each of Go, Rust, but it's not proritized yet.